### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode

### ISSUE THREAD

**Issue by iwasnobody** [Yelp/elastalert#1863]
Any idea?
```
INFO:elastalert:Queried rule authcenter service from 2018-08-27 09:09 UTC to 2018-08-27 09:19 UTC: 274 / 274 hits
INFO:elastalert:Queried rule authcenter service from 2018-08-27 09:19 UTC to 2018-08-27 09:20 UTC: 0 / 0 hits
ERROR:root:Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/elastalert/elastalert.py", line 1321, in alert
    return self.send_alert(matches, rule, alert_time=alert_time, retried=retried)
  File "/usr/local/lib/python2.7/site-packages/elastalert/elastalert.py", line 1410, in send_alert
    alert.alert(matches)
  File "/usr/local/lib/python2.7/site-packages/elastalert/alerts.py", line 1403, in alert
    body = body[0:4000] + "\n⚠ *message was cropped according to telegram limits!* ⚠"
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1: ordinal not in range(128)

ERROR:root:Uncaught exception running rule authcenter service: 'ascii' codec can't decode byte 0xe2 in position 1: ordinal not in range(128)
INFO:elastalert:Rule authcenter service disabled
INFO:elastalert:Ignoring match for silenced rule authcenter service
INFO:elastalert:Ignoring match for silenced rule authcenter service
```

**Comment by Qmando** [comment]
I haven't released a version with this change included yet.

**Comment by Qmando** [comment]
I've just released version 0.1.34, which includes this fix.
